spec/requests/review_pages_spec.rb

```diff
@@ -391,7 +391,7 @@
           end
 
         it "should show the coachee(s)' review table" do
-          page.should have_selector('h1', text: "Viewing All Reviews")
+          page.should have_selector('h1', text: "Watched Reviews")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.